beacon-chain/rpc/prysm/v1alpha1/validator/proposer_execution_payload.go

```diff
@@ -102,10 +102,10 @@ func (vs *Server) getExecutionPayload(ctx context.Context, slot types.Slot, vIdx
 	case true:
 		feeRecipient = recipient
 	case errors.As(err, kv.ErrNotFoundFeeRecipient): // If fee recipient is not found, use the default fee recipient.
-		logrus.WithError(err).WithFields(logrus.Fields{
+		logrus.WithFields(logrus.Fields{
 			"validatorIndex":      vIdx,
 			"defaultFeeRecipient": feeRecipient,
-		}).Error("Fee recipient not found. Using default fee recipient.")
+		}).Error("Fee recipient not found. Using default fee recipient")
```

Comment:
Should this be error level at all? I would think it is more of a WARN level issue.